glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/giantswarm/azure-operator
 import:
 - package: github.com/giantswarm/azuretpr
-  version: master
+  version: resource-group
```

Comment:
Are we going with TPRs still here? Do we have to migrate here to CRDs as well?

Reply:
We're going with TPR. I don't want to bother with vendor hacks, and update operatorkit first. Migration is cheap as long as we don't have customers we can burn all clusters to the ground.

Reply:
Ack. Should I sort the migration then or do you guys want to sort it?

Reply:
If you want to do it you're welcome. Otherwise I'll do that. But let's first finish with this PR to avoid merging conflicts.

Reply:
It would take some workload off of me when you sort this here.

Reply:
Ok, don't worry then.